smart-contract-verifier/tests/middleware_types.rs

```diff
@@ -0,0 +1,153 @@
+pub mod solidity {
```

Comment:
don't quite understand how this relates to middleware testing

is it just functions for generating requests, which can be used not only for middleware testing?